modules/react_paragraphs_behaviors/src/Config/ConfigOverrides.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\react_paragraphs_behaviors\Config;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\react_paragraphs_behaviors\ReactBehaviorsPluginManagerInterface;
+
+/**
+ * Class ConfigOverrides.
+ *
+ * @package Drupal\react_behaviors\Config
+ */
+class ConfigOverrides implements ConfigFactoryOverrideInterface {
+
+  /**
+   * Behavior yaml plugin manager for behavior derivatives.
+   *
+   * @var \Drupal\react_paragraphs_behaviors\ReactBehaviorsPluginManagerInterface
+   */
+  protected $pluginManager;
+
+  /**
+   * Keyed array of behavior derivative definitions.
+   *
+   * @var array
+   */
+  protected $behaviors = [];
+
+  /**
+   * ConfigOverrides constructor.
+   *
+   * @param \Drupal\react_paragraphs_behaviors\ReactBehaviorsPluginManagerInterface $plugin_manager
+   *   Behavior plugin manager service.
+   */
+  public function __construct(ReactBehaviorsPluginManagerInterface $plugin_manager) {
+    $this->pluginManager = $plugin_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
```

Comment:
Should this also expand to allow paragraph_rows?

Reply:
I was debating that. I wasn't sure if it would reduce the scope of this work much.

Reply:
Get to it later then?

Reply:
Was literally just getting it in there haha.

Reply:
I got the overrides to work and changed the logic for automatically enabling.
```
paragraphs:
  - card
rows: {}
```
will NOT enable the behavior plugin for any rows. I can't think of a way to make a behavior apply to _all_ paragraphs or rows. Any thoughts on that?

Reply:
Perhaps something like the ds_fields syntax where they parse a syntax like
`entity|bundle` or `entity|*`. 

Key in the yaml could be:
```
entities:
  - 'paragraph|wysiwyg'
  - 'paragraph|card'
  - 'paragraph_row'
```

Reply:
ooh that could work. a little more tricky to work out but I'll see what I can do

Reply:
The last commit should make that possible. Thanks for the idea.
```
  bundles:
    - paragraphs_type|card
    - paragraphs_type|text
    - paragraphs_row_type|*
```